scripts/kind-e2e/e2e.sh

```diff
@@ -5,8 +5,7 @@ source $(git rev-parse --show-toplevel)/scripts/lib/debug_functions
 
 ### Variables ###
 
-declare -A cluster_CIDRs=( ["cluster1"]="10.244.0.0/16" ["cluster2"]="10.245.0.0/16" ["cluster3"]="10.246.0.0/16" )
-declare -A service_CIDRs=( ["cluster1"]="100.94.0.0/16" ["cluster2"]="100.95.0.0/16" ["cluster3"]="100.96.0.0/16" )
+PRJ_ROOT=$(git rev-parse --show-toplevel)
```

Comment:
you don't need to calculate that, there's a variable in dapper for the prj root.

Reply:
Nice, done in the latest, thanks!